sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/DigitalTwinsLifecycleAsyncSample.java

```diff
@@ -333,4 +332,48 @@ public static void connectTwinsTogether() throws IOException, InterruptedExcepti
         // Wait until the latch count reaches zero, signifying that the async calls have completed successfully.
         connectTwinsLatch.await(MAX_WAIT_TIME_ASYNC_OPERATIONS_IN_SECONDS, TimeUnit.SECONDS);
     }
+
+    /**
+     *
+     * Queries for all the digital twins.
+     * @throws IOException If an I/O error is thrown when accessing the starting file.
+     * @throws InterruptedException If the current thread is interrupted while waiting to acquire latch.
+     */
+    public static void queryTwins() throws IOException, InterruptedException {
```

Comment:
Are IOExceptions actually thrown? I don't see any of our client methods declaring it.

Reply:
Nope. IOException is not needed for query. Will remove it.